### PR TITLE
`connect`: add `\shortcuts` command

### PR DESCRIPTION
**Pull request by sssciel**
Add command `\shortcuts` listing all available shortcuts and hotkeys in go-prompt.

Closes https://github.com/tarantool/tt/issues/541

### CODE REVIEW

**Comment by sssciel** [comment]
image

**Comment by 0x501D** [comment]
Thanks for the patch. Please add an integration test for the feature.

File: cli/connect/shortcuts.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+       Ctrl + H [Backspace]        -- Delete character before the cursor
+       Ctrl + I [Tab]              -- Get next completion
+       BackTab                     -- Get previous completion
+       Ctrl + D                    -- Delete character under the cursor
+       Ctrl + W                    -- Cut the word before the cursor to the clipboard
+       Ctrl + K                    -- Cut the line after the cursor to the clipboard
+       Ctrl + U                    -- Cut the line before the cursor to the clipboard
+       Ctrl + L                    -- Clear the screen
+       Ctrl + R                    -- Enter in the reverse search mode
+...`
```
Comment (askalt) [inline cli/connect/shortcuts.go:26]: Please, take a look at https://github.com/tarantool/tt/blob/master/cli/connect/console.go#L330-L355

Comment (sssciel) [reply cli/connect/shortcuts.go]: fixed

Comment (askalt) [reply cli/connect/shortcuts.go]: And also at keybinds: https://github.com/tarantool/tt/blob/master/cli/connect/console.go#L330-L341. It seems that not everything has been added.

Comment (sssciel) [reply cli/connect/shortcuts.go]: fixed

**Review by 0x501D — APPROVED** [review]

File: cli/connect/shortcuts.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+// getShortcutsList is a command for a get shortcuts and hotkeys list.
+const getShortcutsList = "\\shortcuts"
+
+const shortcutListText = `---
+- - |
+    Available hotkeys and shortcuts:
+
+       Ctrl + J / Ctrl + M [Enter] -- Enter the command
+       Ctrl + A [Home]             -- Go to the beginning of the line
```
Comment (askalt) [inline cli/connect/shortcuts.go:11]: In multi-line command case this moves to the beginning of the command, not line. The same with `Ctrl+E`.
```suggestion
       Ctrl + A [Home]             -- Go to the beginning of the command
```

File: cli/connect/shortcuts.go [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+       Ctrl + E [End]              -- Go to the end of the line
+       Ctrl + P [Up Arrow]         -- Previous command
+       Ctrl + N [Down Arrow]       -- Next command
+       Ctrl + F [Right Arrow]      -- Forward one character
+       Ctrl + B [Left Arrow]       -- Backward one character
+       Ctrl + H [Backspace]        -- Delete character before the cursor
+       Ctrl + I [Tab]              -- Get next completion
+       BackTab                     -- Get previous completion
+       Ctrl + D                    -- Delete character under the cursor
+       Ctrl + W                    -- Cut the word before the cursor to the clipboard
```
Comment (askalt) [inline cli/connect/shortcuts.go:21]: Does it really works with clipboard?

File: cli/connect/shortcuts.go [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+       Ctrl + K                    -- Cut the line after the cursor to the clipboard
+       Ctrl + U                    -- Cut the line before the cursor to the clipboard
+       Ctrl + L                    -- Clear the screen
+       Ctrl + R                    -- Enter in the reverse search mode
+       Ctrl + C                    -- Interrupt current unfinished expression
+       ESC + B                     -- Move backwards one word
```
Comment (askalt) [inline cli/connect/shortcuts.go:27]: Does `Esc + B`/`Esc + F` really works?  
```suggestion
       Alt + B                     -- Move backwards one word
```

File: cli/connect/shortcuts.go (hunk 2 above)
Comment (sssciel) [reply cli/connect/shortcuts.go]: fixed

File: cli/connect/shortcuts.go (hunk 3 above)
Comment (sssciel) [reply cli/connect/shortcuts.go]: fixed, opened issue #562

File: cli/connect/shortcuts.go (hunk 4 above)
Comment (sssciel) [reply cli/connect/shortcuts.go]: fixed

**Review by askalt — APPROVED** [review]

**Comment by psergee** [comment]
Please, use "connect:" prefix for commit/PR title instead of "tt connect:".

File: test/integration/connect/test_connect.py [hunk 5]
```diff
@@ -143,6 +143,55 @@ def skip_if_tarantool_ce():
         pytest.skip("Tarantool Enterprise required")
 
 
+def test_connect_and_get_shorcuts(tt_cmd, tmpdir_with_cfg):
```
Comment (psergee) [inline test/integration/connect/test_connect.py:146]:
```suggestion
def test_connect_and_get_shortcuts(tt_cmd, tmpdir_with_cfg):
```

Comment (sssciel) [reply test/integration/connect/test_connect.py]: fixed

**Pull request merged**